### PR TITLE
Make project structure more generic.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: marketpulse/settings/base.py [hunk 1]
```diff
@@ -98,7 +103,8 @@
 MEDIA_ROOT = config('MEDIA_ROOT', default=os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = config('MEDIA_URL', '/media/')
 
-SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', default=not DEBUG, cast=bool)
+SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', default=not DEBUG,
+                               cast=bool)
```
Comment (johngian) [inline marketpulse/settings/base.py]: There is no need to break this line. Our flake8 config allows max 100 columns.

File: marketpulse/main/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+
+urlpatterns = patterns(
+    url(r'^$', '{0}.main.views'.format(settings.PROJECT_NAME), name='main'),
```
Comment (johngian) [inline marketpulse/main/urls.py]: This should be `{0}.views.main`.

File: marketpulse/settings/base.py [hunk 3]
```diff
@@ -47,6 +43,15 @@
...
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    # Third party apps
+    'django_nose',
+
+    # Project specific apps
+    '%s.base' % PROJECT_MODULE,
+    '%s.main' % PROJECT_MODULE,
+
```
Comment (johngian) [inline marketpulse/settings/base.py]: Too many empty lines here.

**Comment by akatsoulas** [comment]
@johngian pr updated. thanks for the review!

**Comment by johngian** [comment]
~~I suggest we split the configuration to more files in order to separate generic settings with custom ones. My main point here is that we want the toolkit users to build on top of it and not hack the our generic setup.~~

**Comment by johngian** [comment]
r+

**Pull request merged**